web-ui/styles/openclaw-structured.css

```
@@ -27,63 +27,152 @@
+}
+
+.accordion-panel.active .accordion-chevron {
+    transform: rotate(180deg);
 }
 
+.accordion-content {
+    padding: 0 16px 16px;
+    animation: accordionSlide 0.2s ease;
+}
+
+@keyframes accordionSlide {
```

Stylelint (17.12.0)

[error] Expected keyframe name "accordionSlide" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)